### PR TITLE
repo-wide: Fix monitoring.yml files

### PR DESCRIPTION
**Pull request by TheGreatDefector**
Updated yml files, replacing "rss: null" with "rss: ~"

Resolves https://github.com/getsolus/packages/issues/4425

### CODE REVIEW

**Comment by davidjharder** [comment]
Hey, welcome to Solus! I changed your PR description so that the issue will be automatically closed.

File: packages/w/wireless-regdb/monitoring.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 releases:
   id: 15257 # Check https://release-monitoring.org/
```
Comment (davidjharder) [inline packages/w/wireless-regdb/monitoring.yml:2]: For this file 15257 is correct, so please delete the rest of the line.

I can't find an rss feed, so leave `rss: ~` and delete the rest of the line

File: packages/l/lzip/monitoring.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 releases:
   id: 1866 # Check https://release-monitoring.org/
```
Comment (davidjharder) [inline packages/l/lzip/monitoring.yml:2]: 1866 is correct, so the rest of the line can be deleted

**Review by davidjharder — CHANGES_REQUESTED** [review]
A few small corrections.

Also, please amend your commit title and PR title to something like `repo-wide: Fix monitoring.yml files` This is how we style commits that affect more than one package. If it was just one package, then it would be `packagename: Fix monitoring.yml`

**Comment by TheGreatDefector** [comment]
Sorry for the delay, PR and commit renamed now for clarity. Thanks!

**Pull request merged**